strongbox-web-core/src/main/java/org/carlspring/strongbox/controllers/layout/npm/NpmUser.java

```diff
@@ -0,0 +1,109 @@
+package org.carlspring.strongbox.controllers.layout.npm;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonSetter;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class NpmUser
```

Comment:
@steve-todorov @fuss86  can you guys please advice shouldn't this class be placed in `strongbox-web-forms`?

Reply:
Hm maybe. I was following how it was done for LoginInput.java.

Reply:
@ptirador : Perhaps you might have some thoughts on this as well...?

Reply:
> Hm maybe. I was following how it was done for LoginInput.java.

ok, let it be the same in this case